### PR TITLE
Code Housekeeping

### PR DESCRIPTION
**Pull request by vkbo**
Various code cleanups

* Symplecticity deviation is now written to sim_meta.dat instead of its own file
* Removed trailing spaces in all fortran files (again)
* Removed some unnecessary variables
* Added flush statements to fort.93 writes (and a few other places)
* Moved around a lot of file open calls to where they're needed as well as wrapped in conditions. This reduces the number of zero size files in the simulation folder.

### CODE REVIEW

File: source/lielib.f90 [hunk 1]
```diff
@@ -2697,16 +2697,9 @@ subroutine mapflol(sa,sai,cr,cm,st)
...
-        call f_requestUnit('symplecticity_check.txt', symplecticity_check_unit)
-        open(symplecticity_check_unit,file="symplecticity_check.txt", status='replace')
-        write(symplecticity_check_unit,*) c1e2*(xsu-nd2)/xsu
-        close(symplecticity_check_unit)
-#ifdef DEBUG
-!       call warr('symplcdev',100.d0*(xsu-nd2)/xsu,0,0,0,0)
-#endif
+        ! Report
+        meta_sympCheck = c1e2*(xsu-nd2)/xsu
+        write(lout,"(a,es13.6,a)") "LIELIB> Deviation for symplecticity = ",c1e2*(xsu-nd2)/xsu," %"
```
Comment (amereghe) [inline source/lielib.f90]: maybe write `meta_sympCheck` directly

File: source/close.f90 [hunk 2]
```diff
@@ -28,7 +28,7 @@ subroutine closeUnits
   ! Then iterate through the first 1000 units
   do chkUnit=1, units_maxUnit
     ! Do not close the following units:
-    if(chkUnit == 6 .or. chkUnit >= 91 .and. chkUnit <= 97) cycle
+    if(chkUnit == 5 .or. chkUnit == 6 .or. chkUnit >= 91 .and. chkUnit <= 97) cycle
```
[the remark this answers is not in the document]
Comment (vkbo) [reply source/close.f90]: No need to add brackets. That is how it's parsed already.
Unit 5 is stdin.

File: source/lielib.f90 (hunk 1 above)
Comment (vkbo) [reply source/lielib.f90]: Done

File: doc/user_manual/apxInputOutput.tex [hunk 3]
```diff
@@ -25,7 +25,7 @@ \chapter{Input and Output Files} \label{Files}
     \hline
     4 & & \checkmark & Ascii & Geometry and strength Parameters (format as file \texttt{fort.2}) \\
     \hline
-    6 & & \checkmark & Ascii & Input Parameters and Analysis of Data \\
+    6 & & \checkmark & Ascii & Simulation Output \\
```
[the remark this answers is not in the document]
Comment (vkbo) [reply doc/user_manual/apxInputOutput.tex]: Changed it to Standard Output

**Comment by JamesMolson** [comment]
This pr broke the fluka coupling

**Comment by vkbo** [comment]
Can you be a bit more specific?

**Comment by JamesMolson** [comment]
fluka.f90, L485,486

**Comment by amereghe** [comment]
my bad, I overlooked it, thanks, James, for catching it

**Pull request merged**